e2e/cypress/integration/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const iamVersion = (<string><Object>Cypress.env('IAM_VERSION') ?
```

Comment:
😕 I don't understand this `<string><Object>` cast. What's behind this?

Reply:
I'm not sure tbh. I was just factoring it out of the common places we have it.